notebook/agentchat_function_call_currency_calculator.ipynb

```diff
@@ -77,7 +75,7 @@
    "id": "92fde41f",
    "metadata": {},
    "source": [
-    "It first looks for environment variable \"OAI_CONFIG_LIST\" which needs to be a valid json string. If that variable is not found, it then looks for a json file named \"OAI_CONFIG_LIST\". It filters the configs by models (you can filter by other keys as well). Only the models with matching names are kept in the list based on the filter condition.\n",
+    "It first looks for environment variable \"OAI_CONFIG_LIST\" which needs to be a valid json string. If that variable is not found, it then looks for a json file named \"OAI_CONFIG_LIST\". It filters the configs by tags (you can filter by other keys as well). Only the configs with matching tags are kept in the list based on the filter condition.\n",
```

Comment:
We should probably add tags to the example config list below in L82 of the notebook

Reply:
good suggestion. added.